lib/Contracts/IMailManager.php

```diff
@@ -126,4 +126,14 @@ public function markFolderAsRead(Account $account, string $folderId): void;
 	 * @throws ServiceException
 	 */
 	public function flagMessage(Account $account, string $mailbox, int $uid, string $flag, bool $value): void;
+
+	/**
+	 * @param Account $account
+	 * @param string $mailbox
+	 * @param bool $subscribed
+	 *
+	 * @throws ClientException
+	 * @throws ServiceException
+	 */
+	public function subscribeFolder(Account $account, string $mailbox, bool $subscribed): bool;
```

Comment:
```suggestion
	public function updateSubscription(Account $account, string $mailbox, bool $subscribed): bool;
```

Again, rather cosmetic than a functional change. But this name makes it clear that both *subscribe* and *unsubscribe* are handled there. `subscribeFolder` sounds a bit to specific to me.

Reply:
I alway struggle with names;-) Will change it